blockchain/vm/doc.go

```diff
@@ -20,8 +20,11 @@
 
 /*
 Package vm implements the Ethereum Virtual Machine.
-The vm package implements one EVM, a byte code VM. The BC (Byte Code) VM loops
-over a set of bytes and executes them according to the set of rules defined
-in the Ethereum yellow paper.
+
+The vm package implements one EVM, a byte code VM.
+The Byte Code VM loops over a set of bytes and executes them according to the set of rules defined in the Ethereum yellow paper.
+
+As well as the original functionality of the EVM, this package implemented additional pre-compiled contracts to support the native features of Klaytn.
+For more information of pre-compiled contracts, see KlaytnDocs (https://docs.klaytn.com/smart-contract/precompiled-contracts).
```

Comment:
Grammarly suggested.

```suggestion
For more information about pre-compiled contracts, see KlaytnDocs (https://docs.klaytn.com/smart-contract/precompiled-contracts).
```